src/System.Device.Gpio/System/Device/Gpio/Drivers/LibgpiodDriver.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    public class LibgpiodDriver : GpioDriver
+    {
+        private const string GpioGet = "gpioget";
+        private const string GpioChip0 = "gpiochip0 ";
+        private const string GpioSet = "gpioset";
+        public int TimeoutMilliSeconds { get; set; } = (int)TimeSpan.FromSeconds(1).TotalMilliseconds;
+        private class MutableTuple<Item1, Item2>
+        {
+            public Item1 Value1 { get; set; }
+            public Item2 Value2 { get; set; }
+            public MutableTuple(Item1 item1, Item2 item2) {
+                Value1 = item1;
+                Value2 = item2;
+            }
+        }
+        private Dictionary<int, MutableTuple<PinMode, Process>> _pinNumberToPinModeAndProcess = new Dictionary<int, MutableTuple<PinMode, Process>>();
+        protected internal override int PinCount => throw new NotImplementedException();
+
+        protected internal override void AddCallbackForPinValueChangedEvent(int pinNumber, PinEventTypes eventType, PinChangeEventHandler callback)
+        {
+            throw new NotImplementedException();
```

Comment:
in the one above, you used `=>` notation.

Reply:
Well the method signature is longer, don't fit in one row. Anyways which one is preferred/suggested?

Reply:
I guess I wasn't consistent in my feedback 😄 
I think in general
1. We often use `=>` (aka expression bodied member) when the expression is simple and short. Otherwise not. Hopefully we are consistent within a single file.
1. We keep the width of files reasonable to avoid wrapping. Perhaps 200 characters, or less.

Reply:
got it thanks